docs/Data/Maybe/instance/Filter.md

```diff
@@ -0,0 +1,50 @@
+# `Maybe<A>#Filter(Predicate<A> predicate)`
+
+Tests the underlying value against the `predicate`, returning the `Just` of the value for `true`; otherwise, `Nothing`.
+
+## Alternatives
+
+* `Maybe.Filter<A>(Predicate<A> predicate)(IMaybe<A> maybe)`
+* `Maybe.Filter<A>(Predicate<A> predicate, IMaybe<A> maybe)`
+
+## Types
+
+* `A`: The underlying type.
+
+## Arguments
+
+* `predicate (Predicate<A>)`: The predicate with which to test the value.
+
+## Returns
+
+* `(IMaybe<A>)`: The `Just` of the value for `true`; otherwise, `Nothing`.
+
+## Throws
+
+* `ArgumentNullException` if the `predicate` is `null`.
+
+## Examples
+
+```csharp
+Predicate<int> isEven = value => value % 2 == 0;
+Maybe.Just(0)
+  .Filter(isEven);
+// => Just(1)
```

Comment:
This example return doesn't match its input.

Reply:
Ooo… Good catch. (^_^)

Reply:
Fixed in `11d646d`